flowmachine/flowmachine/core/connection.py

```diff
@@ -204,6 +208,40 @@ def _available_dates(self) -> Dict[str, List[datetime.date]]:
             ),
         )
 
+    @property
+    def available_qa_checks(self) -> List[dict]:
+        with self.engine.begin() as trans:
+            return [
+                row._mapping
+                for row in trans.exec_driver_sql(
+                    "SELECT cdr_type, type_of_query_or_check FROM etl.deduped_post_etl_queries GROUP BY cdr_type, type_of_query_or_check"
+                ).fetchall()
+            ]
+
+    def get_qa_checks(self, cdr_type, start_date, end_date, check_type) -> List[dict]:
```

Comment:
Doc and types. Also wondering whether it is worth piping the dates through `utils.parse_datestring` and then formatting to YYYY-MM-dd.